pkg/resolver/searchComplete.go

```diff
@@ -44,13 +45,21 @@ func SearchComplete(ctx context.Context, property string, srchInput *model.Searc
 	if userDataErr != nil {
 		return []*string{}, userDataErr
 	}
+
+	//check that shared cache has resource datatypes:
+	propTypesCache, err := rbac.GetCache().GetPropertyTypes(ctx)
+	if err != nil {
+		klog.Warningf("Error creating datatype map with err: [%s] ", err)
```

Comment:
What happens if we can't get the propertyTypes? Should we "fall back" to only strings?